spec/models/collection_spec.rb

```diff
@@ -55,14 +55,15 @@
     end
 
     context "when adding members" do
-      let(:work1) { create(:work) }
-      let(:work2) { create(:work) }
-      let(:work3) { create(:work) }
-
-      it "allows multiple files to be added" do
-        collection.add_member_objects [work1.id, work2.id]
-        collection.save!
-        expect(collection.reload.member_objects).to match_array [work1, work2]
+      let(:work1) { valkyrie_create(:hyrax_work, title: 'Work 1') }
+      let(:work2) { valkyrie_create(:hyrax_work, title: 'Work 2') }
+      let(:work3) { valkyrie_create(:hyrax_work, title: 'Work 3') }
```

Comment:
Everywhere CollectionMemberService adding members is called in the code, it calls `add_members_by_ids`.  Since this turns around and instantiates the members before calling `add_members`, the tests save a step and just pass in members.  The code is actually adds the members expects a `Hyrax::Resource`.